### ISSUE TITLE
can't edit space description

### ISSUE THREAD

**Issue by riccardoklinger** [heremaps/xyz-qgis-plugin#18]
I've tried to edit a space in QGIS. Unfortunately it throws a warning:
  XYZ Hub : ChainInterrupt(TypeError("QNetworkAccessManager.sendCustomRequest(QNetworkRequest, QByteArray, QIODevice data=None): argument 3 has unexpected type 'bytes'",), 0, '1/2 in chain', )
The edits were not pushed to XYZ.

**Comment by minff** [comment]
Can you show the version of library used in QGIS (found in Help > About section), e.g.
```
QGIS version
3.4.4-Madeira
QGIS code revision
f6ddc62fdb
Compiled against Qt
5.11.2
Running against Qt
5.11.2
Compiled against GDAL/OGR
2.4.0
Running against GDAL/OGR
2.4.0
Compiled against GEOS
3.7.0-CAPI-1.11.0
Running against GEOS
3.7.0-CAPI-1.11.0 673b9939
PostgreSQL Client Version
9.2.4
SpatiaLite Version
4.3.0
QWT Version
6.1.3
QScintilla2 Version
2.10.8
Compiled against PROJ
520
Running against PROJ
5.2.0

```

**Comment by riccardoklinger** [comment]
you asked, I deliver ;-)

QGIS version
3.4.4-Madeira
QGIS code revision
f6ddc62
Compiled against Qt
5.5.1
Running against Qt
5.5.1
Compiled against GDAL/OGR
2.2.2
Running against GDAL/OGR
2.2.2
Compiled against GEOS
3.5.1-CAPI-1.9.1
Running against GEOS
3.5.1-CAPI-1.9.1 r4246
PostgreSQL Client Version
9.5.2
SpatiaLite Version
4.3.0a
QWT Version
6.1.2
QScintilla2 Version
2.9.1
PROJ.4 Version
492

**Comment by minff** [comment]
The fix is added to [version 1.6.2](/heremaps/xyz-qgis-plugin/releases/tag/1.6.2-alpha).
